pages/events/series.vue

```diff
@@ -0,0 +1,105 @@
+<script setup>
+// COMPONENTS
```

Comment:
@jendiamond Make a new route pages/series and create index.vue instead of series.vue